modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonadDeferLaws.kt

```diff
@@ -1,15 +1,9 @@
 package arrow.test.laws
 
 import arrow.Kind
-import arrow.core.Either
-import arrow.core.Left
-import arrow.core.Right
-import arrow.core.Try
-import arrow.core.left
-import arrow.core.recover
-import arrow.core.right
-import arrow.data.k
+import arrow.core.*
```

Comment:
I actually prefer to use start imports, but I thought I ask, what's the project standard?

Reply:
@pablisco After #1405 we had to either ignore all wildcard imports or fix them, we ended up fixing them.

Reply:
I have this setup on my IDE, what's the rule now?